modules/oe_content_sub_entity/src/Event/SubEntityEvent.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_content_sub_entity\Event;
+
+use Drupal\Core\Entity\ContentEntityInterface;
+use Symfony\Component\EventDispatcher\Event;
+
+/**
+ * Event used to form sub-entity type labels individually for each bundle.
+ */
+class SubEntityEvent extends Event {
+
+  /**
+   * The name of the event.
+   */
+  const LABEL_FORMATION = 'oe_content.event.subentity_label_formation';
```

Comment:
```suggestion
  const LABEL_FORMATION = 'oe_content_sub_entity.label_formation';
```
There is a pattern when event is described in the special final class to protect event, please check build/core/modules/block_content/src/BlockContentEvents.php, build/core/lib/Drupal/Core/Session/AccountEvents.php, build/core/modules/config_translation/src/Event/ConfigTranslationEvents.php.